draft-ietf-httpbis-http2bis.xml

```diff
@@ -1032,7 +1032,8 @@
             type <xref target="PROTOCOL_ERROR" format="none">PROTOCOL_ERROR</xref>.
           </t>
           <t>
-            The first use of a new stream identifier implicitly closes all streams in the "idle"
+            The first use of a new stream identifier in a <xref target="HEADERS" format="none">HEADERS</xref>
+            or <xref target="PUSH_PROMISE" format="none">PUSH_PROMISE</xref> frame implicitly closes all streams in the "idle"
             state that might have been initiated by that peer with a lower-valued stream identifier.
```

Comment:
I've a little trouble with the ambiguity regarding stream identifiers in PUSH_PROMISE.  I'm not clear on this, but it might be better to say:

> Each type of stream transitions out of the "idle" state for one reason:
> * A client-initiated stream becomes "open" when a HEADERS frame is sent on the stream.
> * A server-initiated stream becomes "reserved" when a PUSH_PROMISE (sent on a request stream) promises a response on the stream.
> When a stream transitions out of the "idle" state, all streams in that direction with lower-numbered stream identifiers that are also "idle" are closed.  That is, an endpoint can skip a stream identifier, with the effect being that the skipped stream is immediately closed.

Too large a change?

Reply:
No, I think that's a good change. I may try to call out the specific stream identifier in the PUSH_PROMISE frame payload that is relevant here.

Reply:
Hmm, I abandoned that change a bit because it didn't read well with the paragraph above, so I've tried something else. Let me know how you feel about it.